spread.yaml

```diff
@@ -58,6 +58,10 @@ backends:
                 image: ubuntu-16.04-64
                 workers: 6
 
+            - fedora-27-64:
+                image: fedora-cloud-base-27-1-6-x86-64
```

Comment:
Curious, what is the pattern there? 27 is the Fedora release. What about the rest?